frontend/src/Sandbox/Counter.css

```diff
@@ -0,0 +1,54 @@
+.counter-container {
+  touch-action: none;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  /* outline: 1px solid white; */
+  width: 115px;
+  height: 75px;
+  display: inline-block;
+  margin-top: 5px;
+  margin-right: 12px;
+  position: relative;
+  left: 10px;
+}
+
+.counter {
+  background: linear-gradient(
+    to top,
+    rgba(204, 228, 244, 0) 0%,
+    #cce4f4 50%,
+    rgba(204, 228, 244, 0) 100%
+  );
+  border-radius: 5px 5px;
+  border-style: solid;
+  border-color: #02273f;
+  outline: #314a5b;
+  height: 70px;
+  width: 110px;
+  color: #02273f;
+  text-shadow: none;
+  font-size: 28px;
+  text-align: center;
+  /* background-color: #cce4f4; */
+  font-weight: bold;
+  font-family: museo_sans, 'Lucida Grande', 'Lucida Sans', 'Helvetica',
```

Comment:
Doesn't look like the museo_sans is loading by default. It is available on google fonts so lets import if from there.